flow/metrics.yaml

```diff
@@ -19,6 +19,11 @@ components:
           type: boolean
           default: false
           x-field-uid: 2
+        tx_rx_replication_factor:
```

Comment:
I didn't follow what the user accomplishes by setting this configuration?  In the example provided, I don't see how it got used and how it impacted the statistics that were gathered and displayed.

Reply:
I have updated the example (how this is used). Also refer to mail reply with Alex's example.